### PR TITLE
Update readme

### LINKED ISSUE #49: add getting started section to README

### PR DESCRIPTION
**Pull request by NickSchimek**
# Description
Added details about the project layout and renamed the contributing section to Getting Started.

Fixes #49

## Type of changes
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,26 @@ This site is built with Jekyll, a simple, easy to use static site generator.
...
+The main page has different sections. Each section has its own file for html, and assets (css, images, js).
+
+For example lets look at the Products section. The relevant html and asset files for the products section can be found here:
+
+- HTML file: `sections/products.html`
+- CSS file: `assets/stylesheets/products.scss`
+- Images dir: `assets/images/products/*`
+- Javascript dir: `assets/javascript/products/*`
  
+Products is a collection and each individual product can be found in its collection directory. Jekyll will look for the products collection in the directory `_products` by default. Inside that directory you'll find each product and the data that is used to build the secion.
```
Comment (NickSchimek) [inline README.md]:
```suggestion
Products is a collection and each individual product can be found in its collection directory. Jekyll will look for the products collection in the directory `_products` by default. Inside that directory you'll find each product and the data that is used to build the section.
```

File: README.md [hunk 2]
```diff
@@ -11,15 +11,26 @@ This site is built with Jekyll, a simple, easy to use static site generator.
...
-## Contributing
+## Getting Started
 
  - Please read and abide by our [Code of Conduct](https://github.com/codeforpdx/codeofconduct)
 
  - To learn about our workflow and how to get started. View the [CONTRIBUTING.MD](./CONTRIBUTING.md) doc.
 
 ## Project Layout
- - Coming soonish, however the layout is standard Jekyll stuff.
+ - File Organization is similar to any other project built with Jekyll with the following organization for html and asset files
```
Comment (NickSchimek) [inline README.md]:
```suggestion
 - File Organization is similar to any other project built with Jekyll with the following organization for HTML and asset files.
```

File: README.md [hunk 3]
```diff
@@ -11,15 +11,26 @@ This site is built with Jekyll, a simple, easy to use static site generator.
...
+
+The main page has different sections. Each section has its own file for html, and assets (css, images, js).
```
Comment (NickSchimek) [inline README.md]:
```suggestion
The main page has different sections. Each section has its own file for HTML, and assets (CSS, images, JS).
```

File: README.md [hunk 4]
```diff
@@ -11,15 +11,26 @@ This site is built with Jekyll, a simple, easy to use static site generator.
...
+
+For example lets look at the Products section. The relevant html and asset files for the products section can be found here:
```
Comment (NickSchimek) [inline README.md]:
```suggestion
For example lets look at the Products section. The relevant HTML and asset files for the products section can be found here:
```

**Comment by NickSchimek** [comment]
@katrinasperry Let me know if there is something specific missing here that needs to be added.

**Comment by katrinasperry** [comment]
@NickSchimek it looks great, thank youuuuuu!

**Pull request merged**